### PR TITLE
Add new unittest utilities

### PR DESCRIPTION
**Pull request by jedie**

### CODE REVIEW

File: bx_py_utils/test_utils/html_assertment.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from django.contrib.messages import get_messages
+
+
+class HtmlAssertmentMixin:
```
Comment (phihag) [inline bx_py_utils/test_utils/html_assertment.py:4]: Assertment is a very old-school legal term. Likely _assertion_?

File: bx_py_utils/test_utils/html_assertment.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    """
+    Unittest mixin class with usefull assertments around Django test client tests
```
Comment (phihag) [inline bx_py_utils/test_utils/html_assertment.py:6]: typo: useful

File: bx_py_utils/test_utils/html_assertment.py (hunk 1 above)
Comment (jedie) [reply bx_py_utils/test_utils/html_assertment.py]: renamed

File: bx_py_utils/test_utils/html_assertment.py (hunk 2 above)
Comment (jedie) [reply bx_py_utils/test_utils/html_assertment.py]: fixed

**Pull request merged**